scripts/workers/code_upload_worker_utils/monitor_submission.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+echo "Monitoring submission..."
+time_elapsed=0
+while [ $time_elapsed -le $SUBMISSION_TIME_LIMIT ]
+do
+    echo "Checking for submission file after $time_elapsed secs.."
+    sh /evalai_scripts/make_submission.sh
+    if [ -f "$SUBMISSION_PATH/completed.txt" ]
```

Comment:
Instead of this we could've just checked for `submission.csv/json` file in the monitor script. We woudn't need a `completed.txt` in that case. Any reason you preferred this setup?

Reply:
@Ram81 Yes, because I thought monitor_submission.sh should be independent of particulars of what file is submitted. `make_submission.sh` would take care of the particulars. Even if we modify that we want these types of files or maybe in future we want to submit multiple files all that logic would go into `make_submission.sh` and `monitor_submision.sh` would only check for `completed.txt`. In this way we don't have to repeat our logic again and again.

Reply:
Makes sense.